src/app/web-app.component.ts

```diff
@@ -45,15 +53,22 @@ export class WebAppComponent implements OnInit {
    * @param {ThemeStorageService} themeStorageService Theme Storage Service.
    * @param {MatSnackBar} snackBar Material Snackbar for notifications.
    * @param {AlertService} alertService Alert Service.
+   * @param {KeyboardShortcutsService} keyboardShortcuts KeyboardS Shortcuts Service.
+   * @param {SearchService} searchSevice Searchbar Visibility variable service
    */
```

Comment:
The comment should come before the constructor. Also make sure everything being used is mentioned in the comment.